eppoclient/configresponse.go

```diff
@@ -3,11 +3,17 @@ package eppoclient
 import (
 	"encoding/json"
 	"fmt"
+	"strconv"
 	"time"
 
 	semver "github.com/Masterminds/semver/v3"
 )
 
+type AssignmentValue struct {
+	Parsed interface{}
+	Raw    []byte
+}
```

Comment:
This structure doesn't make sense for anything but JSON.

It also **increases** memory allocation for all cases as `Raw` allocates memory on heap (as its size is unknown). Which is quite wasteful as that value is discarded for all non-json assignment.

Reply:
Yea 😮‍💨 should not have worked on this at night and waited to hear your feedback. Thanks for checking it out carefully.